torch/csrc/distributed/c10d/init.cpp

```diff
@@ -632,7 +632,11 @@ Example::
     >>> store = dist.TCPStore("127.0.0.1", 0, 1, True, timedelta(seconds=30))
     >>> # This will throw an exception after 10 seconds
     >>> store.wait(["bad_key"], timedelta(seconds=10))
-)");
+)")
+      .def_property_readonly(
+            "timeout",
+            &::c10d::Store::getTimeout,
+            R"(Gets the default timeout of the store.)");
```

Comment:
nit: can remove "default" since the timeout can be modified by user with [set_timeout](https://pytorch.org/docs/master/distributed.html#torch.distributed.Store.set_timeout)

Reply:
Makes sense. I will remove it in the next revision.